doc/stdlib/arrays.rst

```diff
@@ -637,13 +637,29 @@ BitArrays
 
    Performs a bitwise not operation on B. See :ref:`~ operator <~>`.
 
+.. function:: rol!(dest::BitArray{1},src::BitArray{1},i::Integer) -> BitArray{1}
+
+   Performs a left rotation operation on dest.
```

Comment:
This is not very clear, even if t's self-evident from the argument names the description should state that the rotation is performed on `src` and put into `dest`.

Also a very minor nitpick: please put a space between the function arguments after the commas - here and below.